docs/conf.py

```diff
@@ -12,59 +12,63 @@
 #
 import os
 import sys
-sys.path.insert(0, os.path.abspath('.'))
+
+sys.path.insert(0, os.path.abspath("."))
```

Comment:
The project was previously standardized on single quotes, and PEP8 allows one to make either choice. I'd prefer not to make this change, since it will be widespread for little benefit in readability.